src/main/java/core/basesyntax/WorkWithFile.java

```
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String buyer = "buy";
+        String supplier = "supply";
+        int buy = 0;
+        int supply = 0;
 
+        File file = new File(fromFileName);
+        try {
+            BufferedReader readLine = new BufferedReader(new FileReader(file));
+            String value = readLine.readLine();
+            while (value != null) {
+                if (value.contains(supplier)) {
+                    int index = value.indexOf(",") + 1;
+                    String name = value.substring(index, value.length());
+                    supply += Integer.parseInt(name);
+                } else if (value.contains(buyer)) {
+                    int index = value.indexOf(",") + 1;
+                    String name = value.substring(index, value.length());
+                    buy += Integer.parseInt(name);
```

Integer parsing is done directly on the substring result (e.g., `Integer.parseInt(name)`) without validating the CSV parts or wrapping parsing errors. Per the checklist guidance to not ignore errors ("Don't ignore exceptions"), invalid number formats should surface with a descriptive RuntimeException including the offending line and the original exception as the cause.